templates/legal/terms-and-policies/discourse-terms.md

```diff
@@ -0,0 +1,76 @@
+---
+wrapper_template: "legal/_base_legal_markdown.html"
+context:
+  title: "Canonical Ubuntu Community Discourse Forum Terms of Service"
+  description: "Ubuntu and Canonical Legal - Canonical Ubuntu Community Discourse Forum Terms of Service"
+  update_date: "Version - April 2023"
+  copydoc: https://docs.google.com/document/d/1Yv4bItPlNPS4oOSvUsUavT3qD6NmVrf_EjQNVFLx40w/edit
+---
+
+# Ubuntu Community Discourse Forum terms and conditions
+
+## Canonical's terms and conditions
+
+These terms and conditions cover your registration and use of the Ubuntu Community Discourse Forum (“**Forum**”), provided by Canonical Group Limited, registered in England, company number 06870835 (“**Canonical**”, “**us**”, “**we**” or “**our**”) to you as an individual or an entity (“**you**” or “**your**”). Your registration and use of the Forum will be governed by these terms and conditions along with Canonical's Privacy Policy and Privacy Notice. Please read these terms and conditions carefully before you register for a Forum account. If you register for a Forum account and use the Forum on behalf of a company or other entity, you represent that you have the full legal authority to bind the company to these terms and conditions and you are agreeing to these terms and conditions on behalf of that company. To register for a Forum account and use the Forum, you must be at least 13 years old. If you are between age 13 and 18, you confirm that you have your parent's or legal guardian's consent and that they have read and agreed to these terms.
```

Comment:
Would it be better to change `“` to `"`?

Reply:
I am not sure, maybe but these days browsers tend to have the entire character set.